### DISCUSSION TITLE
Better/Streamlined Meta Quest Pipeline

### DISCUSSION THREAD

**Discussion by isaaciverson** [godotengine/godot-proposals#4005] [Platforms]
Considering the mass adoption and use of the Meta Quest platform, it may be advantageous for Godot to provide more comprehensive support for it.

**Comment by clayjohn** [comment]
You'll be very interested to see the recent grants we have received from meta to do just that! https://godotengine.org/article/godot-engine-receiving-new-grant-meta-reality-labs
